docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,59 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
     ```
 
+- **Normal element**
+
+    All elements in the pipeline have specific properties and they can be manipulated to control the operation of a pipeline. To get and set the property value, you have to get the element handle in the pipeline by calling `ml_pipeline_element_get_handle()` with its name.
+    ```c
+    ml_pipeline_h handle = nullptr;
+    ml_pipeline_element_h demux_h = nullptr;
+
+    pipeline = g_strdup("videotestsrc ! video/x-raw,format=RGB,width=640,height=480 ! videorate max-rate=1 ! " \
+	    "tensor_converter ! tensor_mux ! tensor_demux name=demux ! tensor_sink");
+
+    // Construct a pipeline
+    status = ml_pipeline_construct (pipeline, NULL, NULL, &handle);
+
+    // Get the handle of target element
+    status = ml_pipeline_element_get_handle (handle, "demux", &demux_h);
+    ```
+
+    After fetching the handle of the target element, you can set and get the value of the specific property.
+    ```c
+    gchar *ret_tensorpick;
+
+    // Set the string value of given element's property
+    status = ml_pipeline_element_set_property_string (demux_h, "tensorpick", "1,2");
+
+    // Get the string value of given element's property
+    status = ml_pipeline_element_get_property_string (demux_h, "tensorpick", &ret_tensorpick);
+    ```
+
+    Before you destroy the pipeline, release `demux_h` by calling `ml_pipeline_element_release_handle()`.
+    ```c
+    ml_pipeline_element_release_handle (demux_h);
+    ```
+
+    To figure out the property information of the target element, you can run `gst-inspect-1.0` command on your device as below.
+    ```bash
+    # gst-inspect-1.0 tensor_demux
+    ...
+    Element Properties:
+    name                : The name of the object
+                            flags: readable, writable
+                            String. Default: "tensordemux0"
+    parent              : The parent of the object
+                            flags: readable, writable
+                            Object of type "GstObject"
+    silent              : Produce verbose output ?
+                            flags: readable, writable
+                            Boolean. Default: true
+    tensorpick          : Choose nth tensor among tensors ?
```

Comment:
- Why is there a? mark 

   Produces verbose output.

Also, can you please format text under element properties?

Reply:
Well, that is a just help message when running `gst-inspect-1.0 tensor_demux` command.
I don't know why the owner adds the question mark but it is not necessary anyway.
Since this filter is made by us so I will update the help message ASAP and remove it on this page.